### PR TITLE
fix: [BT-2970]: Update README.md

### PR DESCRIPTION
**Pull request by AnshuH**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>

You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger utAll`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/44604)

### CODE REVIEW

**Review by AnshuH — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/44604)* status: 1 file(s) to review. 0 of 1 approvals obtained. 

___
*[`README.md` line 7 at r1](https://reviewable.io/reviews/harness/harness-core/44604#-NPWll4W6gGLM46KXMBz:-NPWll4W6gGLM46KXMC-:bim52vg) ([raw file](https://github.com/harness/harness-core/blob/0759189a2eae0dd8d8592935891b2c20c03d9e8d/README.md#L7)):*
> ```Markdown
> ## On MacOS
> ```

This is invalid

**Comment by diptiman-adak** [comment]
*[`README.md` line 8 at r1](https://reviewable.io/reviews/harness/harness-core/44604#-NPWm2LxCavhfIt5izqL:-NPWm2LxCavhfIt5izqM:bb74njr) ([raw file](https://github.com/harness/harness-core/blob/0759189a2eae0dd8d8592935891b2c20c03d9e8d/README.md#L8)):*
> ```Markdown
> 
> 
> #### Please go through the following wiki page to setup local environment to build harness-core with bazel. Minimal and required steps are written in below wiki page.
> ```

Test

**Review by diptiman-adak — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/44604)* status: 1 file(s) to review. 1 change requests. 0 of 1 approvals obtained. 

___
*[`README.md` line 7 at r1](https://reviewable.io/reviews/harness/harness-core/44604#-NPWm-xW7nQzH-abXrwv:-NPWm-xW7nQzH-abXrww:b-80x5a7) ([raw file](https://github.com/harness/harness-core/blob/0759189a2eae0dd8d8592935891b2c20c03d9e8d/README.md#L7)):*
> ```Markdown
> ## On MacOS
> ```

Space

**Review by AnshuH — COMMENTED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/44604#-:-NPWmM_v7WRIHGQxjchJ:bnfp4nl)* status: 1 file(s) to review. 1 change requests. 1. unresolved discussions. 0 of 1 approvals obtained. 

___
*[`README.md` line 7 at r1](https://reviewable.io/reviews/harness/harness-core/44604#-NPWm-xW7nQzH-abXrwv:-NPWmQ33-k7Ssn4iBO_U:b32wtd) ([raw file](https://github.com/harness/harness-core/blob/0759189a2eae0dd8d8592935891b2c20c03d9e8d/README.md#L7)):*
<details><summary><i>Previously, diptiman-adak (Diptiman Adak) wrote…</i></summary><blockquote>

Space
</blockquote></details>

ok
___
*[`README.md` line 9 at r1](https://reviewable.io/reviews/harness/harness-core/44604#-NPWm8fh0YioZesDiFf6:-NPWm8fh0YioZesDiFf7:b32vzs) ([raw file](https://github.com/harness/harness-core/blob/0759189a2eae0dd8d8592935891b2c20c03d9e8d/README.md#L9)):*
> ```Markdown
> 
> #### Please go through the following wiki page to setup local environment to build harness-core with bazel. Minimal and required steps are written in below wiki page.
> #### You can ignore steps 1 to 3 in "Prerequisities" section and can ignore steps 1 to 3 in "Build" section after following the below wiki.
> ```

NA

**Review by diptiman-adak — COMMENTED** [review]
aaa

**Comment by AnshuH** [comment]
trigger messagecheck

**Review by AnshuH — COMMENTED** [review]
:lgtm:

> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/44604#-:-NQ-pgjEC93X8EYc2zl3:bnfp4nl)* status: Some of the checks are too old. 1 file(s) to review. 1 change requests. 2. unresolved discussions. 0 of 1 approvals obtained. 

___
*[`README.md` line 8 at r1](https://reviewable.io/reviews/harness/harness-core/44604#-NPWm2LxCavhfIt5izqL:-NQ-qUAE6nds3A1U38C2:b32wtd) ([raw file](https://github.com/harness/harness-core/blob/0759189a2eae0dd8d8592935891b2c20c03d9e8d/README.md#L8)):*
<details><summary><i>Previously, diptiman-adak (Diptiman Adak) wrote…</i></summary><blockquote>

Test
</blockquote></details>

ok

**Review by AnshuH — COMMENTED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/44604#-:-NQ-qdXlE2ZIAD-xCzCK:bnfp4nl)* status: Some of the checks are too old. 1 file(s) to review. 1 change requests. 2. unresolved discussions. 0 of 1 approvals obtained.

**Review by AnshuH — COMMENTED** [review]
test

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ harness-core Project Dev environment setup instructions
 ======================================================
 ## On MacOS
 
+
```
Comment (AnshuH) [inline README.md:7]: Hi

**Pull request closed without merging**